relay-filter/src/health_check.rs

```diff
@@ -0,0 +1,162 @@
+//! Implements event filtering based on whether the endpoint called is a healthcheck endpoint
+//!
+//! If this filter is enabled messages to healthcheck endpoints will be filtered out
+
+use once_cell::sync::Lazy;
+use regex::Regex;
+use relay_general::protocol::Event;
+
+use crate::{FilterConfig, FilterStatKey};
+
+static HEALTH_CHECK_ENDPOINTS: Lazy<Regex> = Lazy::new(|| {
+    Regex::new(
+        r#"(?ix)
+        healthcheck|
+        healthy|
+        live|
+        ready|
+        heartbeat|
+        /health$|
+        /healthz$
+        "#,
+    )
+    .expect("Invalid healthcheck filter Regex")
+});
+
+fn matches(_event: &Event) -> bool {
```

Comment:
I think this is a cargo warning?
```suggestion
fn matches(event: &Event) -> bool {
```

Reply:
yep, remained _event from the initial commit when it was just a skeleton not using the event